src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,207 @@
+.header {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+
+  &__logo {
+    padding: 0 50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+}
```

Comment:
This block also should be in separate file.

`11.[FILE STRUCTURE] - You should create separate file for each BEM block`

The same about other blocks.

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)